src/client/Microsoft.Identity.Client/ApiConfig/Executors/PublicClientExecutor.cs

```diff
@@ -114,46 +111,8 @@ public PublicClientExecutor(IServiceBundle serviceBundle, PublicClientApplicatio
             return await handler.RunAsync(cancellationToken).ConfigureAwait(false);
         }
 
-        private IWebUI CreateWebAuthenticationDialog(
```

Comment:
This code got moved to the Orchestrator, since it is responsible for creating dependencies (broker, webui )